### PR TITLE
added google finance option and ftehc method

### PR DESCRIPTION
**Pull request by joe-pritchard**
Added method for google finance, and command line option -g, or --google

### CODE REVIEW

**Comment by joe-pritchard** [comment]
@Torann , have a chance to look at this? I have something else I'd like to add once this is merged :)

File: src/Console/Update.php [hunk 1]
```diff
@@ -105,6 +114,35 @@ private function updateFromOpenExchangeRates($defaultCurrency, $api)
...
+        $this->info('Updating currency exchange rates from finance.google.com...');
+        foreach ($this->currency->getDriver()->all() as $code => $value) {
+            // don't update the default currency, the value is always 1
+            if ($code === $defaultCurrency) {
+                continue;
+            }
+
+            $response = $this->request('http://finance.google.com/finance/converter?a=1&from=' . $defaultCurrency . '&to=' . $code);
+
+            if (\Illuminate\Support\Str::contains($response, 'bld>')) {
```
Comment (Torann) [inline src/Console/Update.php]: Move the class to `use Illuminate\Support\Str;` at the top of the file

File: src/Console/Update.php [hunk 2]
```diff
@@ -105,6 +114,35 @@ private function updateFromOpenExchangeRates($defaultCurrency, $api)
...
+                $data     = explode('bld>', $response);
+                $rate     = explode($code, $data[1])[0];
+                $this->currency->getDriver()->update($code, [
+                    'exchange_rate' => $rate,
+                ]);
+            } else {
```
Comment (Torann) [inline src/Console/Update.php]: New line after `}` so it is

```
} 
else {
```

**Comment by Torann** [comment]
Looks good! Left comments about some super minor tweaks.

Google isn't dropping this anytime soon are they?

**Comment by joe-pritchard** [comment]
Completely forgot about this, sorry. Changes done as requested :)

I can't see Google removing finance any time soon I think its quite widely used...

**Comment by joe-pritchard** [comment]
Hi mate - just a bump; I'm actively using the google finance option in my project and I'd like to get back on the main fork :)

**Comment by Torann** [comment]
Looks good!

**Pull request merged**